lib/modules/manager/terraform/modules.spec.ts

```diff
@@ -115,4 +116,67 @@ describe('modules/manager/terraform/modules', () => {
       expect(dots.tag).toBe('v1.0.0');
     });
   });
+
+  describe('azureDevOpsSshRefMatchRegex', () => {
+    it('should split organization, project, repository and tag from source url', () => {
+      const ssh = azureDevOpsSshRefMatchRegex.exec(
+        'git@ssh.dev.azure.com:v3/MyOrg/MyProject/MyRepository?ref=1.0.0'
+      ).groups;
```

Comment:
```suggestion
      )?.groups;
```

typescript null check